packages/components/psammead-episode-list/src/fixtures.jsx

```diff
@@ -206,7 +201,7 @@ export const renderEpisodes = ({
               <EpisodeList.Metadata>
                 <>
                   {' '}
-                  <StyledSpan aria-hidden>|</StyledSpan> {episode.date}
+                  <EpisodeList.VerticalSeparator /> {episode.date}
                 </>
               </EpisodeList.Metadata>
```

Comment:
I don't think we need the spaces here any more given the `VerticalSeperator` has its own spacing (from the padding and margin)

The spaces seem to be mis-aligning the divider: top is with spaces, bottom without:
![Screen Shot 2020-12-11 at 10 07 42](https://user-images.githubusercontent.com/8601460/101890726-fd8ee400-3b98-11eb-819e-c3de6121db9e.png)